code/unit_tests/atmospherics_tests.dm

```diff
@@ -197,7 +197,7 @@
 	name = "ATMOS MACHINERY: scrub_gas() Conserves Moles"
 
 /datum/unit_test/atmos_machinery/conserve_moles/scrub_gas/start_test()
-	var/list/filtering = subtypesof(/decl/material/gas)
+	var/list/filtering = decls_repository.get_decls_of_subtype(/decl/material/gas)
```

Comment:
`get_decl_paths_of_subtype()` would return a typepath instead of an instance, which I think is what scrub_gas on line 206 wants.